__tests__/distributors/microsoft-installer.test.ts

```diff
@@ -0,0 +1,88 @@
+import { MicrosoftDistributions } from '../../src/distributions/microsoft/installer';
+
+describe('findPackageForDownload', () => {
+  let distribution: MicrosoftDistributions;
+
+  beforeEach(() => {
+    distribution = new MicrosoftDistributions({
+      version: '',
+      architecture: 'x64',
+      packageType: 'jdk',
+      checkLatest: false
+    });
+  });
+
+  it.each([
+    [
+      '17.x',
+      '17.0.1',
+      'https://aka.ms/download-jdk/microsoft-jdk-17.0.1.12.1-{{OS_TYPE}}-x64.{{ARCHIVE_TYPE}}'
+    ],
+    [
+      '16.0.x',
```

Comment:
Given 16 is not an LTS release of MS Build of OpenJDK, I think we should not list it here.

Reply:
as above, but if you feel strongly I will remove it.

Reply:
No, it's fine. See previous comment.